cc/tiles/tile_manager.cc

```diff
@@ -135,9 +135,24 @@ class RasterTaskImpl : public TileTask {
     if (std::string("renderer") ==
         base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII("type")) {
       ResourcePool::SoftwareBacking* sw_backing = resource_.software_backing();
-      if (sw_backing)
-        mojo::SyncSharedMemoryHandle(sw_backing->SharedMemoryGuid(),
-            0, resource_.size().width() * resource_.size().height() * 4);
+      if (sw_backing &&
+          UsePartialMemorySync(content_rect_, invalid_content_rect_)) {
+        size_t offset_x = (invalid_content_rect_.x() - content_rect_.x()) * 4;
```

Comment:
Instead of 4, 
int bytes_per_pixel = BitsPerPixel(resource_.format()) / 8;

Reply:
Thanks, done.